polynote-frontend/polynote/notification/notifications.ts

```diff
@@ -12,16 +13,17 @@ export class NotificationHandler {
 
     private enabled: boolean = false;
     private constructor() {
+        super()
         const handlePref = (pref: typeof UserPreferences["notifications"]) => {
-            if (pref.value) {
+            if (pref.value !== undefined) {
```

Comment:
This has a probably unintended effect – it will always ask you to enable notifications when you run a cell.

Reply:
Thanks for the catch!

Reply:
fixed in https://github.com/polynote/polynote/pull/1016/commits/c9eb374b73b70d9bede22305c502d160622210f9